arachni/Makefile

```diff
@@ -0,0 +1,4 @@
+NAME := arachni
+VERSION: = 1.0.0
```

Comment:
I caught it during testing but haven't submitted.

Maybe worth discussing versioning some time? k8s doesn't like to constantly deploy `latest` but I'm not sure how that ties to auto builds on docker hub.